src/main/java/com/minecolonies/coremod/client/gui/WindowHutLumberjack.java

```diff
@@ -85,6 +92,26 @@ public WindowHutLumberjack(final BuildingLumberjack.View building)
         pullLevelsFromHut();
     }
 
+
+    /**
+     * Method to send the message to switch the toggle to the server, then updates button
+     */
+    private void switchReplant()
+    {
+
+        ownBuilding.shouldReplant = !ownBuilding.shouldReplant;
+        MineColonies.getNetwork().sendToServer(new LumberjackReplantSaplingToggleMessage(building, ownBuilding.shouldReplant));
+        updateReplantButton();
+    }
+
+    private void updateReplantButton()
+    {
+        Button buttonReplant = findPaneOfTypeByID(BUTTON_TOGGLE_REPLANT, Button.class);
+
+        if (ownBuilding.shouldReplant) {buttonReplant.setLabel(LanguageHandler.format(TOGGLE_REPLANT_SAPLINGS_ON));}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00122)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)